includes/settings/settings.php

```diff
@@ -103,10 +103,7 @@ public function register_admin_menu() {
 	}
 
 	public function display_home_screen() {
-		$editor_preferences = SettingsManager::get_settings_managers( 'editorPreferences' );
```

Comment:
We realized that WordPress doesn't support the dark mode. Therefore we shouldn't activate it on this page, because it will look odd.